tests/system/action/user/test_forget_password_confirm.py

```diff
@@ -0,0 +1,30 @@
+from urllib.parse import quote
+
+from tests.system.action.base import BaseActionTestCase
+
+
+class UserForgetPasswordConfirm(BaseActionTestCase):
```

Comment:
Add a test with an invalid token and assert that the request fails.

Reply:
I added a test and improved the code. I added a needed try/except block.